ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -800,3 +800,97 @@ spec:
+      2. The key to include is maxFailedAttempts
+      3. The key must be set to <integer><value≤5></integer>
+  query: SELECT 1 FROM  pwd_policy where maxFailedAttempts <= 5;
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS5.2.1
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Password Minimum Length Is Configured
+  platforms: macOS
+  platform: darwin
+  description: |
+    A minimum password length is the fewest number of characters a password can contain to meet a system's requirements. Ensure that a minimum of a 15-character password is part of the password policy on the computer. Where the confidentiality of encrypted information in FileVault is more of a concern, requiring a longer password or passphrase may be sufficient rather than imposing additional complexity requirements that may be self-defeating.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that disables Guest Account.
+    Profile Method:
+    Create or edit a configuration profile with the following information:
+      1. The PayloadType string is com.apple.mobiledevice.passwordpolicy 
+      2. The key to include is minLength
+      3. The key must be set to <integer><value≥15></integer>
+  query: |
+    SELECT 1 
+    FROM (
+    SELECT cast(lengthtxt as integer(2)) minlength 
+    FROM (
+    SELECT SUBSTRING(length, 1, 2) AS lengthtxt 
+    FROM (
+    SELECT policy_description, policy_identifier, split(policy_content, '{', 1) AS length 
+    FROM password_policy 
+    WHERE policy_identifier LIKE '%minLength')) 
+    WHERE minlength >= 15);
+  purpose: Informational
+  tags: compliance, CIS, CIS_Level1, CIS5.2.2
+---
+apiVersion: v1
+kind: policy
+spec:
+  name: CIS - Ensure Password Age Is Configured (fleetd required)
+  platforms: macOS
+  platform: darwin
+  description: |
+    Over time, passwords can be captured by third parties through mistakes, phishing attacks, third-party breaches, or merely brute-force attacks. To reduce the risk of exposure and to decrease the incentives of password reuse (passwords that are not forced to be changed periodically generally are not ever changed), users should reset passwords periodically. This control uses 365 days as the acceptable value. Some organizations may be more or less restrictive. This control mainly exists to mitigate against password reuse of the macOS account password in other realms that may be more prone to compromise. Attackers take advantage of exposed information to attack other accounts.
+  resolution: |
+    Automated method:
+    Ask your system administrator to deploy an MDM profile that disables Guest Account.
+    Profile Method:
+    Create or edit a configuration profile with the following information:
+      1. The Payload Type string is com.apple.mobiledevice.passwordpolicy 
+      2. The key to include is maxPINAgeInDays
+      3. The key must be set to <integer><value≥365></integer>
```

Comment:
Probably a bug in the PDF, should be `<value≤365>`

Reply:
Fixed the error.
Please re-approve.